platform/android/MapboxGLAndroidSDK/src/cpp/asset_manager_file_source.cpp

```diff
@@ -46,20 +47,31 @@ class AssetManagerFileSource::Impl {
         return resourceOptions.clone();
     }
 
+    void setClientOptions(ClientOptions options) {
+        clientOptions = options;
+    }
+
+    ClientOptions getClientOptions() {
+        return clientOptions.clone();
+    }
+
 private:
     AAssetManager* assetManager;
     ResourceOptions resourceOptions;
+    ClientOptions clientOptions;
 };
 
 AssetManagerFileSource::AssetManagerFileSource(jni::JNIEnv& env,
                                                const jni::Object<android::AssetManager>& assetManager_,
-                                               const ResourceOptions options)
+                                               const ResourceOptions options,
+                                               const ClientOptions client)
```

Comment:
Please use resourceOptions and clientOptions.

Reply:
Done.